codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -310,14 +315,27 @@ update these file paths accordingly.
         entitlementsFile.absolute.path
       ],
     ];
-    final io.ProcessResult result = await processManager.run(args);
-    if (result.exitCode != 0) {
-      throw CodesignException(
+
+    io.ProcessResult? result;
+    while (retryCount > 0) {
+      result = await processManager.run(args);
```

Comment:
can we log before we run these args what we're attempting? I think it's helpful even if it succeeds to see what the script is doing. Then it's also obvious that we're re-trying.

Reply:
Sure, added a log.

Reply:
I meant before line 323 (thus inside the while loop):

```
log.info('Executing: $args');
```

Reply:
This will allow us to catch, for example, if codesigning succeeds but we accidentally signed with the wrong certificate name.

Reply:
moved log to inside the loop. Ah thanks for explaining Christopher!